next/src/components/sections/ArticlesSection/ArticlesAll.tsx

```diff
@@ -56,36 +60,38 @@ const ArticlesAllSection = ({ section }: Props) => {
   }
 
   return (
-    <div className="flex flex-col gap-8">
+    <div className="flex flex-col">
       <ArticlesFilter
         pageCategories={pageCategoriesData?.pageCategories?.data ?? []}
         tags={tagsData?.tags?.data ?? []}
         onTagChange={handleTagsChange}
       />
-      {title || text ? (
-        <div className="flex flex-col gap-2">
-          {title && <Typography variant="h2">{title}</Typography>}
-          {text && <Typography variant="p-default">{text}</Typography>}
+      <SectionHeader
+        title={title}
+        text={text}
+        // TODO Correct spacing between SectionHeader and remaining content
+        className="pb-6 lg:pb-8"
```

Comment:
Isn't it better to keep the gap in the parent div? Why to introduce wrong spacing handling by bottom paddings?